src/main/cli/cli.c

```diff
@@ -989,8 +989,12 @@ static void cliShowArgumentRangeError(const char *cmdName, char *name, int min,
     }
 }
 
-static const char *nextArg(const char *currentArg)
+STATIC_UNIT_TESTED const char *nextArg(const char *currentArg)
 {
+    if (!currentArg) {
```

Comment:
Where do we have cases where `NULL` is passed into `nextArg()`? I think it's probably a better approach to fix these cases.

Reply:
Thats an interesting take on this. Makes sense though, having the function fail fast is easier on debugging.

I'll close the pr

Reply:
@Igorshp: There is a wider take on this - a large number of all functions taking pointer parameters will potentially crash / fail if `NULL` is passed in. Preventing this would require null checks / bail out code in potentially 100s of places in Betaflight, resulting in considerable code bloat.
In software development in general, most of the time the approach is to consider it acceptable for internal function to come with a contract that (explicitly or implicitly) states that `NULL` is not acceptable input, and expect (internal) users of these functions to respect the contract.